lightning/src/offers/offer.rs

```diff
@@ -381,6 +436,44 @@ impl Offer {
 		Ok(InvoiceRequestBuilder::new(self, metadata, payer_id))
```

Comment:
`request_invoice` docs should be updated to: s/Creates an [`InvoiceRequest`]/Creates a builder for an [`InvoiceRequest`], to match `Offer::new` and `Refund::new`